src/context.cpp

```diff
@@ -19,6 +19,10 @@
 #include "environment.h"
 #include "context.h"
 
+#ifdef WITH_TRACKING
```

Comment:
Are you going to allow DLL and not only Static lib compilation as well ? This may introduce discrepancies in that scenario

Reply:
what kind of discrepancies ?
We would like to have both options on the table